Module1.4/exercises/Get_Users/test/get_users_test.js

```diff
@@ -0,0 +1,39 @@
+var expect = require("chai").expect;
+var chai = require("chai");
+var sinon = require("sinon");
+var sinonChai = require("sinon-chai");
+var request = require("request");
+var getUsers = require("../get_users");
+chai.should();
+chai.use(sinonChai);
+
+describe('GetUsers Tests', () => {
+
+    var spy;
+
+    beforeEach(() => {
```

Comment:
I would like you to add comments on what is being done in the beforeEach, afterEach, etc. But in general, I see it well, good job!